build/Targets/Settings.targets

```diff
@@ -150,7 +150,8 @@
 
         <Otherwise>
           <PropertyGroup>
-            <VisualStudioBuildToolsNuGetPackagePath>$(NuGetPackageRoot)\Microsoft.VSSDK.BuildTools\15.0.25604-Preview4</VisualStudioBuildToolsNuGetPackagePath>
+            <!-- TODO: Detect Preview 4/5 versus RC -->
+            <VisualStudioBuildToolsNuGetPackagePath>$(NuGetPackageRoot)\Microsoft.VSSDK.BuildTools\15.0.25825-RC</VisualStudioBuildToolsNuGetPackagePath>
```

Comment:
They don't work because the install step of the build relies on binaries from the installed VS.  You're right though that it looks like I can condition it on the existence of that binary under `$(DevenvDir)`.